pkg/go/v1/Makefile

```diff
@@ -270,11 +277,20 @@ artifacts/archives/$(PROJECT_NAME)-$(GO_APP_VERSION)-windows-%.zip: $(GO_ARCHIVE
 	@rm -f "$@"
 	zip --recurse-paths --junk-paths "$@" -- $^
 
+artifacts/archives/$(PROJECT_NAME)-$(GO_APP_VERSION)-windows-%.zip.sha256: artifacts/archives/$(PROJECT_NAME)-$(GO_APP_VERSION)-windows-%.zip
```

Comment:
Could probably collapse this recipe and the one for other platforms together with a simpler pattern? I might be missing something though, I'm trying to read this diff on my phone 😀

Reply:
Yep, good call. Done.